docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+
+Email BCC address(es) for email notifications.
+
+### ILS_MAIL_ENABLE_TEST_RECIPIENTS (True/False)
+
+Enable or disable sending mail to test recipients.
+
+### ILS_MAIL_NOTIFY_TEST_RECIPIENTS (list)
+
+When `ILS_MAIL_ENABLE_TEST_RECIPIENTS` is True, all emails are sent here.
+
+```python
+ILS_MAIL_NOTIFY_TEST_RECIPIENTS = ["john.doe@example.com"]
+```
+
+## Assets Configuration
+
+### COLLECT_STORAGE
+
+Static files collection method (defaults to copying files).
+
+```python
+COLLECT_STORAGE = "flask_collect.storage.file"
+```
+
+## Accounts Configuration
+
+For accounts configuration, refer to the [flask-security](https://github.com/inveniosoftware/flask-security-fork) and [invenio-accounts](https://invenio-accounts.readthedocs.io/en/latest/) documentation.
+
+Here's the continuation of the comprehensive documentation for the provided Python config file in Markdown format:
+
+## Celery Configuration
+
+### CELERY_BROKER_URL
+
+URL for the Celery message broker (default: RabbitMQ).
+
+### CELERY_RESULT_BACKEND
+
+URL for the Celery result backend (default: Redis).
+
+### CELERY_BEAT_SCHEDULE
+
+Scheduled tasks configuration for Celery (cronjobs).
+
+Example:
+
+```python
+CELERY_BEAT_SCHEDULE = {
+    "indexer": {
+        "task": "invenio_indexer.tasks.process_bulk_queue",
+        "schedule": timedelta(minutes=5),
+    },
+    "send_overdue_loan_reminders": {
+        "task": "invenio_app_ils.circulation.notifications.tasks.send_overdue_loans_notification_reminder",
+        "schedule": crontab(minute=0, hour=2),
+    },
+    "stats-aggregate-events": {
+        "task": "invenio_stats.tasks.aggregate_events",
+        "schedule": timedelta(hours=3),
+        "args": [("record-view-agg", "file-download-agg")],
+    },
+}
+```
+
+## Feature Configurations
+
+### EXTEND_LOANS_LOCATION_UPDATED (True/False)
+
+Flag to indicate whether to start a task to update active loans when closures of a location are updated.
+
+### EXTEND_LOANS_SCHEDULE_TIME
+
+Time at which the task to extend loans should be run.
+
+```python
+EXTEND_LOANS_SCHEDULE_TIME = datetime.time(2)
+```
+
+### ILS_SELF_CHECKOUT_ENABLED (True/False)
+
+Enable or disable Self-Checkout feature endpoint.
+
+## Database Configuration
+
+### SQLALCHEMY_DATABASE_URI
+
+Database URI including user and password.
+
+```python
+SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://test:psw@localhost/ils"
+```
+
+## JSONSchemas Configuration
+
+### JSONSCHEMAS_HOST
+
+Hostname used in URLs for local JSONSchemas.
+
+```python
+JSONSCHEMAS_HOST = "127.0.0.1:5000"
+```
+
+## CSRF Configuration
+
+### REST_CSRF_ENABLED (True/False)
+
+Enable or disable CSRF protection for REST APIs.
+
+## CORS Configuration
+
+### REST_ENABLE_CORS (True/False)
+
+Enable or disable CORS support for REST APIs.
+
+### CORS_SEND_WILDCARD (True/False)
+
+Allow or disallow credentials to be sent with CORS requests.
+
+### CORS_SUPPORTS_CREDENTIALS (True/False)
+
+Exable or disable supporting credentials for CORS requests.
```

Comment:
these are "security variables", we should add a warning that it is not advised to modify them